src/test/java/org/opensrp/repository/postgres/LocationTagRepositoryTest.java

```diff
@@ -128,6 +128,32 @@ public void testAddShouldNotAddDuplicateLocationTagMap() {
 		
 	}
 	
+	@Test
+	public void testShouldDeleteLocationTagMapByLocationId() {
+		LocationTag locationTag1 = initTestLocationTag6();
+		locationTagRepository.add(locationTag1);
+		
+		List<LocationTag> locationTags = locationTagRepository.getAll();
+		LocationTagMap locationTagMap = new LocationTagMap();
+		locationTagMap.setLocationId(1l);
+		locationTagMap.setLocationTagId(locationTags.get(0).getId());
+		
+		locationTagRepository.addLocationTagMap(locationTagMap);
+		List<LocationTagMap> locationTagMaps = locationTagRepository.getLocationTagMapByExample(1l, locationTags.get(0)
```

Comment:
assert that this list has 1 element

Reply:
I have added not equal for delete list and another equal 1 not deleting list